doc/source/devel/howto_macros/macros_general.rst

```diff
@@ -596,12 +596,16 @@ parameters with different *flavors*:
         self.execMacro('mv', [[motor.getName(), '0']])
         self.execMacro('mv', motor.getName(), '0') # backwards compatibility - see note
 
-    * parameters as space separated string (this is not compatible with multiple
-      or nested repeat parameters, furthermore the repeat parameter must be the last one)::
+    * parameters as space separated string::
 
         self.execMacro('ascan %s 0 100 10 0.2' % motor.getName())
-        self.execMacro('mv %s 0' % motor.getName())
-
+        self.execMacro('mv [[%s 0]]' % motor.getName()) 
+        self.execMacro('mv [%s 0]' % motor.getName()) # simplified interface
```

Comment:
After implementing changes agreed in the last #405 comments this should not be accepted. Please remove if you agree.